ios/FluentUI/Tab Bar/SideTabBar.swift

```diff
@@ -0,0 +1,299 @@
+
+    private let backgroundView: UIVisualEffectView = {
+        var style = UIBlurEffect.Style.regular
+        if #available(iOS 13, *) {
+            style = .systemChromeMaterial
+        }
+
+        return UIVisualEffectView(effect: UIBlurEffect(style: style))
+    }()
+
+    private let topStackView: UIStackView = {
+        return createStackView(spacing: Constants.topItemSpacing)
+    }()
+
+    private let bottomStackView: UIStackView = {
+        return createStackView(spacing: Constants.bottomItemSpacing)
+    }()
+
+    private let avatarViewGestureRecognizer: UITapGestureRecognizer = {
+        return UITapGestureRecognizer(target: self, action: #selector(handleAvatarViewTapped))
+    }()
+
+    private func initialize() {
+        translatesAutoresizingMaskIntoConstraints = false
+
+        backgroundView.translatesAutoresizingMaskIntoConstraints = false
+        contain(view: backgroundView)
+
+        borderLine.translatesAutoresizingMaskIntoConstraints = false
+        addSubview(borderLine)
+
+        addSubview(topStackView)
+        addSubview(bottomStackView)
+
+        if #available(iOS 13, *) {
+            addInteraction(UILargeContentViewerInteraction())
+        }
+
+        accessibilityTraits = .tabBar
+
+        NSLayoutConstraint.activate([widthAnchor.constraint(equalToConstant: Constants.viewWidth),
+                                     borderLine.leadingAnchor.constraint(equalTo: trailingAnchor),
+                                     borderLine.bottomAnchor.constraint(equalTo: bottomAnchor),
+                                     borderLine.topAnchor.constraint(equalTo: topAnchor)])
+    }
+
+    private func setupLayoutConstraints() {
+        if layoutConstraints.count > 0 {
+            NSLayoutConstraint.deactivate(layoutConstraints)
+            layoutConstraints.removeAll()
+        }
+
+        if let avatarView = avatarView {
+            layoutConstraints.append(contentsOf: [
+                avatarView.topAnchor.constraint(equalTo: safeAreaLayoutGuide.topAnchor, constant: Constants.avatarViewTopPadding),
+                avatarView.centerXAnchor.constraint(equalTo: centerXAnchor),
+                avatarView.widthAnchor.constraint(equalToConstant: Constants.avatarViewSize),
+                avatarView.heightAnchor.constraint(equalToConstant: Constants.avatarViewSize),
+                topStackView.topAnchor.constraint(equalTo:avatarView.bottomAnchor , constant: Constants.avatarViewTopStackViewPadding)
+            ])
+        } else {
+            layoutConstraints.append(topStackView.topAnchor.constraint(equalTo: safeAreaLayoutGuide.topAnchor, constant: Constants.topStackViewTopPadding))
+        }
+
+        layoutConstraints.append(contentsOf: [
+            topStackView.widthAnchor.constraint(equalTo: widthAnchor),
+            topStackView.heightAnchor.constraint(equalToConstant: SideTabBar.stackViewHeight(topStackView)),
```

Comment:
Unable to find the other comment about this (thought I left one) but can't we achieve this using contentHuggingPriority and contentCompressionResistancePriority so we don't need to come back and update this if we change the factors we use for laying out the UIStackView?

Reply:
Explanation from my other comment:
The TabBarItemView's frame CGRectZero. This is definitely a bug. TabBarItemView contains an image view and a label that are laid out outside of the view's bounds. I tried quickly fixing TabBarItemView but this caused severe layout bugs in TabBarView. If TabBarItemView's root view was sized correctly, we could use contentHuggingPriority.
Fixing TabBarItemView is out of scope for this change and this current logic isn't adding much complexity.
Is there a way to file bugs for the FluentUI repo?

Reply:
We just use standard github issues to track issues. Can you file one and add a reference to it in a comment next to stackViewHeight so someone looking at this code later knows when it can go away?

Reply:
Filed a bug and added a comment.